packages/frontend/src/features/sqlRunner/components/ContentPanel.tsx

```diff
@@ -84,7 +82,7 @@ export const ContentPanel: FC = () => {
     } = useSqlQueryRun({
         onSuccess: (data) => {
             if (data) {
-                dispatch(setSqlRunnerResults(data));
+                dispatch(setSqlRunnerResults({ data, selectedChartType }));
```

Comment:
since 
```
const selectedChartType = useAppSelector(
        (state) => state.sqlRunner.selectedChartType,
    );
```

you can keep the previous dispatch and handle the selectedChartType on the slice... what do you think?
this is a non-blocking suggestion if it works

Reply:
Good question, this is one of the little spots I'm wasn't sure about, but I don't have a great idea how to do this. 

This problem is that this action gets picked up by other slices that don't have selectedChartType. That comes from the sqlRunner slice and the chart slices listen to this action. There might be a better way to do this, but I can't think of it without effort. Can you? We can try to figure something out together if you want.